dask/base.py

```diff
@@ -150,6 +185,44 @@ def compute(*args, **kwargs):
 
 
 def visualize(*args, **kwargs):
```

Comment:
Variadic positional here feels a bit nicer as an api (no need to build a list, and works with single and many arguments trivially), and you can't have named keywords with variadic positional (`def foo(*args, some_name=None):` is a syntax error).